_includes/setup/android-setup.md

```diff
@@ -39,11 +39,13 @@ To prepare to run and test your Flutter app on the Android emulator, follow thes
 
 1. Enable [VM acceleration](https://developer.android.com/studio/run/emulator-acceleration.html) on your machine.
 1. Launch **Android Studio>Tools>Android>AVD Manager** and select **Create Virtual Device**.
+(If the **Tools** menu has no **Android** submenu, see
+[StackOverflow](https://stackoverflow.com/questions/40407124/android-studio-2-2-2-tools-android-menu-is-missing-in-mac-os-x.)
```

Comment:
As mentioned over in the bug I don't think this SO answer is actually what's going on

Reply:
It seemed from that SO post, that it could be one of several things. I'm not sure where to point them.

Reply:
I would just add a note that the menu entry is only present when inside an Android project. As noted in the bug this is all hopefully temporary anyway.